manual/installation.tex

```diff
@@ -349,17 +349,26 @@ \subsection{Role of QMC\_DATA}
 files are not included in the source code distribution to minimize
 size. The tests are activated if CMake detects the files when
 configured. See tests/performance/NiO/README,
-tests/solids/NiO\_afqmc/README, and tests/performance/C-graphite/README
+tests/solids/NiO\_afqmc/README, tests/performance/C-graphite/README, and tests/performance/C-molecule/README
 for details of the current tests and input files and to download them.
 
 Currently the files must be downloaded via\\
 \url{https://anl.box.com/s/pveyyzrc2wuvg5tmxjzzwxeo561vh3r0} and \\
-\url{https://anl.box.com/s/j5d6jeazpgx5441s04ajtv77iogoyjsh}.
+\url{https://anl.box.com/s/j5d6jeazpgx5441s04ajtv77iogoyjsh} (C-graphite file).
 
```

Comment:
Please add a link to C h5 files.

Reply:
See comment below. After updating we should only have a single unique box url.